services/services.go

```diff
@@ -178,7 +184,7 @@ func getRelaysPageData() (*types.RelaysResp, error) {
 					limit 1
 				) as latest_slot
 			from (
-				select * 
+				builder_pubkey
```

Comment:
The `select` statement is still required.

Also, at least the `tag_id` has to be selected too as it is used to group this properly, see
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/4887c3412af6f61c937a9d70d2ab363c6e8324eb/services/services.go#L191

So at least something like this:

```suggestion
				select builder_pubkey, tag_id
```

Reply:
added tag_id